Added documentation for Automated Threat Hunting within Cybersecurity…

smahuzaifa opened the pull request:
Adding Automated Threat Hunting Documentation

rspoerl1 reviewed (changes requested):
Very comprehensive guide. I just have a couple notes:

1. For consistency, change the attribution blocks in each of your .md files to this format: 
:::info
**Document Creation:** <date> **Last Edited:** <date> **Authors:** John Doe.
:::
2. Rename your image files to be more descriptive, as per the guide - ("Are all your images / videos named appropriately and in the correct location ("image1.png") e.g. will not be accepted).
3. Perhaps reduce some of the code blocks, particularly in the Phase 2 doc. 
4. Small thing, but in Phase 4, there is a duplicate paragraph ("This transition marks..."). 
5. In Phase 5, remove the APP_SECRET parameter value.
6. Exclude the yarn.lock deletion from your PR.

smahuzaifa commented:
Made the following changes based on the feedback:

Updated attribution blocks in all .md files to the :::info format for consistency.

Renamed image files to more descriptive names as per the naming guidelines.

Reduced length of some code blocks in the Phase 2 documentation for better readability.

Removed the duplicate paragraph from Phase 4 (“This transition marks...”).

Removed the APP_SECRET value from Phase 5 to avoid exposing sensitive information.

Excluded the deletion of yarn.lock from the pull request.

rspoerl1 commented:
Great. Please also enclose your attributes within an info block.

smahuzaifa commented:
Attributes have been added within an info block. Thanks

smahuzaifa commented:
Will be creating a new PR to solve the issue with yarn.lock
